Quantum-Control-Applications/Superconducting/Single Fixed Transmon/Use Case - icognito - Optimized readout with optimal weights/IQ_blobs_opt_weights_benchmark.py

```diff
@@ -0,0 +1,129 @@
+"""
+Measure the qubit in the ground and excited state to create the IQ blobs. If the separation and the fidelity are good
+enough, gives the parameters needed for active reset
+"""
+from qm.qua import *
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from configuration import *
+from TwoStateDiscriminator import TwoStateDiscriminator
```

Comment:
We would want to import this from `qualang_tools` and not from this folder (to showcase `qualang_tools` and not have people copy paste this file)

Reply:
yes, once we have it I will update it